Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -354,6 +354,10 @@ def create_pull_request(cls, repository, args, branch_point):
                 sys.stderr.write("Failed to create pull-request for '{}'\n".format(repository.branch))
                 return 1
             print("Created '{}'!".format(pr))
+            # avoid circular import
+            from .revert import Revert
+            if Revert.is_revert_commit(commits[0]):
+                Revert.add_comment_to_reverted_commit_bug_tracker(repository, args, commits[0])
```

Comment:
This will result in a double comment, right? Because we are still making the normal comment on line 370. That also makes me wonder if the `PullRequest` program should actually own the `add_comment_to_reverted_commit_bug_tracker` since we already own the re-opening and standard comment.